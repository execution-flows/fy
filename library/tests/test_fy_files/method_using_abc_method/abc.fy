method encrypt(message: str) -> str
