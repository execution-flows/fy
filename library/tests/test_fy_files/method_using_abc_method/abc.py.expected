import abc


class Encrypt_MethodMixin_ABC(abc.ABC):
    @abc.abstractmethod
    def _encrypt(self, message: str) -> str:
        raise NotImplementedError()
